style.css

```diff
@@ -266,3 +266,47 @@ p {
   gap: 8px;
   padding: 0;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  margin: 0 24px;
+}
+
+.flex-item-1 {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.flex-item-2 {
+  align-self: stretch;
+}
```

Comment:
- [ ] Kindly use flexbox to position your elements, as instructed in the project requirements.